config/fixtures.js

```diff
@@ -0,0 +1,93 @@
+import { consensus } from "./contracts/consensus.js";
+import { usernames } from "./contracts/usernames.js";
+import { multiPayment } from "./contracts/multipayment.js";
+
+const fixtureConfig = {
+    "transfer": {
+        recipientAddress: "0x6F0182a0cc707b055322CcF6d4CB6a5Aff1aEb22",
+        value: "100000000",
+    },
+    "transfer-0": {
+        nonce: "123",
```

Comment:
I think we can skip the nonces in all the fixtures. The reason the existing fixtures have them is because the transaction tester was used for it and that would use the current nonce values. Setting it to 1 will suffice for fixture generation